kpi/permissions.py

```diff
@@ -253,6 +253,13 @@ class EditSubmissionPermission(SubmissionPermission):
     }
 
 
+class DuplicateSubmissionPermission(SubmissionPermission):
+    perms_map = {
+        'GET': ['%(app_label)s.view_%(model_name)s'],
```

Comment:
:eyes:

Reply:
Yes, yes it is.

Reply:
Haha unfortunately it failed unless I had a `GET` in there too...